eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/exception/CommentNotFoundException.java

```diff
@@ -0,0 +1,5 @@
+package hu.flowacademy.companycalendar.exception;
+
+public class CommentNotFoundException extends NotFoundException {
+  public CommentNotFoundException(Long id) { super("Cannot find comment with id: " + id); }
```

Comment:
Ne collapse-oljuk össze az egy soros methodokat sem! (IntelliJ-ben be lehet állítani)